dask/dataframe/io/parquet/core.py

```diff
@@ -218,116 +223,25 @@ def read_parquet(
     if meta.index.name is not None:
         index = meta.index.name
 
-    ignore_index_column_intersection = False
-    if columns is None:
-        # User didn't specify columns, so ignore any intersection
-        # of auto-detected values with the index (if necessary)
-        ignore_index_column_intersection = True
-        columns = [c for c in meta.columns]
+    # Aggregate parts/statistics if we are splitting by row-group
+    if statistics:
+        if chunksize is None:
+            chunksize = 250000000
```

Comment:
I propose that by default we don't do any auto-aggregation, but that if the user specifies a number we treat it as a number of bytes, and if they provide a string, like `"256 MiB"` then we parse it with `parse_bytes`

```python
if chunksize is not None:
    chunksize = parse_bytes(chunk_size)
```